apps/src/templates/sectionProgress/standards/StandardsLegendForPrint.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import ProgressBoxForLessonNumber from './ProgressBoxForLessonNumber';
+
+const styles = {
+  key: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  keyItems: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    margin: '0 10px'
+  }
+};
+
+export default class StandardsLegend extends Component {
+  render() {
+    return (
+      <div style={styles.key}>
+        <strong>{i18n.keyWithColon()}</strong>
+        <span style={styles.keyItems}>
+          {i18n.completedLessons()}
+          <ProgressBoxForLessonNumber completed={true} lessonNumber={1} />
+        </span>
+        <span style={styles.keyItems}>
+          {i18n.uncompletedLessons()}
+          <ProgressBoxForLessonNumber completed={false} lessonNumber={1} />
+        </span>
+      </div>
+    );
```

Comment:
Probably more of a product question, but would it be helpful to make the standards view and the printed report more similar so we can re-use more components/strings?